src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_custom.py

```diff
@@ -56,7 +56,7 @@ def test_add_role_assignment_basic(self):
         with mock.patch(
                 'azure.cli.command_modules.acs.custom.create_role_assignment') as create_role_assignment:
             ok = _add_role_assignment(cli_ctx, role, sp, delay=0)
-            create_role_assignment.assert_called_with(cli_ctx, role, sp, scope=None)
+            create_role_assignment.assert_called_with(cli_ctx, role, sp, True, scope=None)
```

Comment:
is it able to add test for `False` scenario?

Reply:
Sure i can do that.

Reply:
Hello. Could we have a test case for `False` scenario?